.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,3 +10,5 @@ Fixes issue #<issue_number>
 
 - no need to update because...
 - updated...
+
+The Doxygen for this PR can be viewed at http://myosin.sourceforge.net/; click the folder whose name is this PR's number.
```

Comment:
Link to `http://myosin.sourceforge.net/?C=N;O=D` (decreasing order) so the folder appears at the top? :elephant: